src/core/utils/code-block.ts

```diff
@@ -0,0 +1,49 @@
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { css, html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+/**
+ *  A simple styled box for showing code-based error messages.
+ *  @slot the full message
+ *
+ */
+@customElement('uui-code-block')
+export class UUICodeBlock extends LitElement {
+	static styles = [
+		UUITextStyles,
+		css`
+			:host {
+				display: block;
+				font-family: monospace;
+			}
+
+			#container {
+				border: 1px solid var(--uui-color-divider-emphasis);
+				color: var(--uui-color-text-alt);
+				background-color: var(--uui-color-divider-standalone);
+				padding: var(--uui-size-space-2);
+				border-radius: var(--uui-border-radius);
+				line-height: var(--uui-size-10);
+			}
+			:host uui-scroll-container {
+				max-height: 500px;
+				overflow-y: auto;
+				overflow-wrap: anywhere;
+			}
+		`,
+	];
+
+	render() {
+		return html`<div id="container">
+			<uui-scroll-container>
+				<slot></slot>
```

Comment:
Where did we land on the `<pre>` and `<code>` fields here?

Reply:
Forgot to sync my changes before I requested a review - but i wrapped it in a <pre>. Normally you add in <code> too for each line of code, but i figured maybe the element may be used for other types of error messages too, not only a code block.  Maybe the element should be renamed?

Reply:
I think the name is fine. I would also just wrap it in `<code></code>` since any dumps of huge texts could be whatever and contain all sorts of stuff.